playbooks/common/openshift-master/config.yml

```diff
@@ -349,7 +349,8 @@
   roles:
   - role: openshift_master_cluster
     when: openshift_master_ha | bool and openshift.master.cluster_method == "pacemaker"
-  - openshift_examples
+  - role: openshift_examples
+    when: openshift.common.openshift_install_examples | bool
   - role: openshift_cluster_metrics
```

Comment:
This should be openshift.common.install_examples since you are setting install_examples as a local_fact for the common role.